content/gsoc/faq.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Frequently Asked Questions"
+---
+
+- **Where can I find the list of project ideas for this year?**  
+Our project ideas can be found [here](https://www.honeynet.org/gsoc/gsoc-2024/google-summer-of-code-2024-project-ideas/)
+
+- **How do I get started?**  
+Read through the list of [ideas](https://www.honeynet.org/gsoc/gsoc-2024/google-summer-of-code-2024-project-ideas/) and identify the project ideas you find interesting. Play around with our tools! Install the project, use it. Have a look at the issue tracker.
+
+- **How do I start contributing to a project?**  
+Often projects have issues tagged which are considered good starting points. Updating documentation is often a low barrier start and gets you familiar with the process of contributing to an open source project.
+
+- **How do I get in contact with the mentors and other students?**  
+Join us on [Discord](https://discord.gg/68B8Ru5fSU) and talk to your potential mentors.
```

Comment:
I think it's useful from a student's perspective to have a single entrypoint where all mentors are present. But of course you can redirect them to your favorite Slack/Zulip/Discord/... from there right away. Does that make sense?

Reply:
Imho it makes sense if this was our first time GSoC as an org and if another environment has not ever existed: but we have a working Slack environment and the transition to Discord is not easy for me... I am losing people in this way if I don't manage 2 different environments at the same time. 
I rase up this problem once you all wanted to move to Discord but I was probably the only one who was using Slack effectively so I got ignored.

Still it's a problem to me and honestly it's quite frustrating if first you ask my opinion on this and then when I write my perspective you merge the PR without considering it :P

Reply:
Why can't we ask everyone in the gsoc slack to move to Discord? I have seen this done w/o problem at others orgs I am part of.

Reply:
I do not have the time or capacity to add yet another platform in my routine. So I would also appreciate it if we can keep slack and then redirect people to Discord if their projects are using that.

Reply:
@mlodic: Sorry, that was miscommunication on my end! I did not mean to end the discussion here by merging. My intent was to merge this PR here as an improvement of the status quo in all other aspects (the "use Discord" advice was already there beforehand), but still work with you on finding a solution.

I see you feel strongly for using Slack for IntelOwl. How do you think we should communicate this? We can add this prominently to your idea descriptions, as well as extend the Discord mentions in the FAQ with a "For IntelOwl, please use Slack instead" disclaimer. The only part I feel somewhat strongly about is that we shouldn't establish Slack as a general alternative entrypoint, because that would mean that all projects are split across two channels. I'm fine with anything else, feel free to just open a PR if you want. :)

Reply:
> "For IntelOwl, please use Slack instead" disclaimer.
That could work for me but it seems it is not only a problem of mine so I don't know if @m-vasiloma has a better suggestion